crypto/s2n_stream_cipher_rc4.c

```diff
@@ -78,6 +79,7 @@ static int s2n_stream_cipher_rc4_set_decryption_key(struct s2n_session_key *key,
 
 static int s2n_stream_cipher_rc4_init(struct s2n_session_key *key)
 {
+    OSSL_PROVIDER_load(NULL, "legacy");
```

Comment:
Should the return value not be checked to confirm the provider loaded correctly?

Reply:
yep, this was the WIP version and then I switch to the config file. Pushed the latest version with checks and all.